IPFS_ALL_HANDS_FORMAT.md

```diff
@@ -45,18 +45,36 @@ If you have a talk you want to give, [sign up here](https://docs.google.com/spre
 ![](https://res.cloudinary.com/blockchain-side-hustle/image/upload/v1542475278/ipfs_calendar_dyxtst.jpg)
 
 
+## Steps to signing up and speaking
 
-_[Sign up here if you would like to speak!](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898)_
+1. Write a short blurb about your talk and a proposed date for your talk. Send it to portia @ protocol dot com. You should receive an update about the status of your talk within 24-48 hours.
 
+2. After your talk has been confirmed, prepare a slides for your talk. Your presentation should be approximately 15-20 minutes long.
 
-## While you're here check out some of our past talks! 
+3. On the day of the talk you will be given a zoom link. Sign on to zoom and be prepared to speak after announcements.
 
-[IPFS ALL Hands Call November 12, 2018](https://youtu.be/0fT9HC2Crqw)
+4. Two days after you've finished presenting, your talk will be emailed. Feel free to help build awareness of IPFS and share your video on social media!
 
-[IPFS ALL Hands Call November 5, 2018](https://youtu.be/4gVUKbXT0cM)
 
-[IPFS ALL Hands Call October 29, 2018](https://youtu.be/o1tUzgThZK0)
+### Topics that We're Looking for: 
 
-[IPFS ALL Hands Call October 22, 2018](https://youtu.be/iEagJfB8AvM)
+* Any direct work with any of the IPFS APIs (i.e. [go-ipfs](https://github.com/ipfs/go-ipfs), [js-ipfs](https://github.com/ipfs/js-ipfs)
 
+* Building technologies that support IPFS, like [IPLD](https://github.com/ipld/js-ipld)
 
+* Applications such as [Peer Star App](https://github.com/ipfs-shipyard/peer-star-app) that is built on top of IPFS
+
+* Distributed applications where IPFS is part of the development stack, see [OpenBazaar](https://openbazaar.org/blog/openbazaar-2018-roadmap/)
+
+* IPFS used to build alternative decentralized applications that [directly address societal problems](https://observer.com/2017/05/turkey-wikipedia-ipfs/))
+
+
+### If you're looking for more inspiration, check out some of our past talks! 
+
+[IPFS ALL Hands Call November 12, 2018](https://youtu.be/0fT9HC2Crqw?t=116)
+
+[IPFS ALL Hands Call November 5, 2018](https://youtu.be/4gVUKbXT0cM?t=52)
+
+[IPFS ALL Hands Call October 29, 2018](https://youtu.be/o1tUzgThZK0?t=18)
+
+[IPFS ALL Hands Call October 22, 2018](https://youtu.be/iEagJfB8AvM?t=124)
```

Comment:
It would be better if these were actual the talk tiles and speakers rather than just saying `IPFS All Hands Call`

Reply:
We didn't start having talks until November. I'm going to leave `IPFS ALL Hands Call` title for Youtube videos that used the old format.